src/desktop/apps/feature/routes.coffee

```diff
@@ -1,14 +1,16 @@
 Feature = require '../../models/feature.coffee'
 
-@index = (req, res) ->
+@index = (req, res, next) ->
   new Feature(id: req.params.id).fetch
     error: res.backboneError
     success: (feature) ->
-      res.locals.sd.FEATURE = feature
-      res.locals.sd.TAB = req.params.tab
-      res.render 'templates/index',
-        feature: feature
+      if feature.get('version') && feature.get('version') > 1
```

Comment:
👍

Reply:
Oh, quick silly thing: I think this should be `feature.has(version')  && feature.get('version') > 1`? — just because I had to think for a moment considering that this value could very well be 0 (by default?) in which case it'd still work — cuz the 0 would be falsey; but just is a bit weird.

Reply:
Ahhh good catch.